src/components/subsetDraft/Classification.jsx

```diff
@@ -272,20 +281,19 @@ export const ClassificationInfo = ({id, info}) => {
         <div style={{backgroundColor: '#eff4f5'}}
              className='panel'>
             <Title size={4}>{t('Code list info')}</Title>
-            <Paragraph><strong>Id:</strong> {id}</Paragraph>
-            <table style={{border: 'none'}}>
-                <thead>
-                <th>{t('From')}</th>
-                <th>{t('To')}</th>
-                <th>{t('Version')}</th>
-                </thead>
+            <Paragraph><strong>Id:</strong> {id}</Paragraph>k
+                <tr>
+                <td>{t('From')}</td>
+                <td>{t('To')}</td>
+                <td>{t('Version')}</td>
+                </tr>
```

Comment:
why?

Reply:
React did not like the table at all here. It did render the table ok even though it crashed. Since we are not displaying a table we do need a table, semantic wise.